modules/main-rules/pages/data-sheets-and-characteristics.adoc

```diff
@@ -57,7 +57,14 @@ Some rules affect some unit types and not others.
 
 Speed:: The distance that you can move the unit in a single movement action (in centimetres).
 It represents a rate at which the unit can move across the battlefield and still be effective when it shoot its weapons.
-You can give units orders that allow you to move units farther than this, but their shooting will suffer.
+You can give units orders that allow you to move units farther than this, but their shooting will suffer. +
++
+Where a unit has a split Speed value such as 20/30 cm, use the first value for all purposes except Assault moves.
```

Comment:
As per the Issue discussion.

```suggestion
Where a unit has a split Speed value such as 20/30 cm, use the first value for all purposes except charging into base contact.
```

Reply:
I think that this is all too much detail for this topic. Remember, this is one of the 'Basics' topics, and while we write in 'every page is page one' topics, some level of 'progressive disclosure' still has value and I think that it is reasonable to assume that newbies will tend to read this long before they get to Assault rules and charge moves and such.

I do think that it is good to acknowledge the split speeds here though, and to reference the section that explains them more. So, I'll edit with this in mind and push an update later then see what you think.

Reply:
> I do think that it is good to acknowledge the split speeds here though, and to reference the section that explains them more. So, I'll edit with this in mind and push an update later then see what you think.

What section are you referring to? The Assault moves section obviously details how the second value works, but you can't move the general explanation of a characteristic there.

Reply:
Outdated, fixed in a newer commit. So I'll resolve this conversation.